string Class
string additional_label
float64 probability
sensor_msgs/Image mask